tox.ini

```diff
@@ -205,7 +205,7 @@ envlist =
 
 [testenv]
 deps =
-  -c dev-requirements.txt
+  -r dev-requirements.txt
```

Comment:
Why was this needed?

Reply:
my bad, it was something while debugging locally, reverted

Reply:
This was the reason for pypy build failures